primitives/rpc/src/tracing.rs

```diff
@@ -88,10 +88,13 @@ pub struct Span {
 #[derive(Serialize, Deserialize, Default, Clone, Debug)]
 pub struct Values {
 	/// HashMap of `bool` values
+	#[serde(skip)]
```

Comment:
I think we should only skip these fields if empty - eg, we currently use a bool to indicate a wasm (vs native) trace.

Reply:
These are only used for serializing the JSON response, correct?

If so, I think we should leave it out for now since I don't think it is particularly helpful to have in the RPC response. (We can use other logging for that?). Seems like extra logic we can always add in later.

Reply:
@mattrutherford do we ever need to `Deserialize` with serde? Maybe we can remove that derive?

Reply:
no we'll never need to, but afaicr it's a requirement of the RPC setup

Reply:
> These are only used for serializing the JSON response, correct?
> 
> If so, I think we should leave it out for now since I don't think it is particularly helpful to have in the RPC response. (We can use other logging for that?). Seems like extra logic we can always add in later.

I think it could be helpful to know if a trace is WASM or native, because there's often no other way to confirm that from the name or target